### PR TITLE
Updating readme for MySQL and PostgreSQL to include new flexible server namespace

### PR DESCRIPTION
**Pull request by mjain2**
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i>

This PR is to update MD files for azure.mgmt.rdbms namespace, which includes mysql and postgresql.

Overview of changes:
- Adding new API version (already checked in) for both MySQL and PostgreSQL to the .md files
- Updates and adding md files for SDKs for Go, .NET and Python (includes multi-api changes)

### Contribution checklist:
- [ ] I commit to follow the [Breaking Change Policy](http://aka.ms/bcforapi) of “no breaking changes
- [ ] I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow.
- [ ] [Validation tools](https://aka.ms/swaggertools) were run on swagger spec(s) and errors have all been fixed in this PR. [How to fix?](https://aka.ms/ci-fix)

If any further question about AME onboarding or validation tools, please view the [FAQ](https://aka.ms/faqinprreview).

### ARM API Review Checklist
- [ ] Ensure to check this box if one of the following scenarios meet updates in the PR, so that label “WaitForARMFeedback” will be added automatically to involve ARM API Review. Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs, all “removals” and “adding a new property” no more require ARM API review.
  - Adding new API(s)
  - Adding a new API version
  - Adding a new service

- [ ] If you are blocked on ARM review and want to get the PR merged with urgency, please get the ARM oncall for reviews (*RP Manifest Approvers* team under <ins>Azure Resource Manager service</ins>) from IcM and reach out to them. 

### Breaking Change Review Checklist 
If there are following updates in the PR, ensure to request an approval from API Review Board as defined in the [Breaking Change Policy](http://aka.ms/bcforapi). 

- [ ] Removing API(s) in stable version
- [ ] Removing properties in stable version
- [ ] Removing API version(s) in stable version
- [ ] Updating API in stable version with Breaking Change Validation errors
- [ ] Updating API(s) in preview over 1 year

Please follow the link to find more details on [PR review process](https://aka.ms/SwaggerPRReview).

### CODE REVIEW

**Comment by mjain2** [comment]
@qianwens 
Hello, we have activated the PR and it is ready for review.  The Python package has breaking changes (swagger APIs are checked in already).  We have added capabilities for mutli api and plan on bumping the Python version from 2.2.0 to 2.3.0 for the release package. Please let me know if you have any questions.

**Review by changlong-liu — APPROVED** [review]
LGTM

File: specification/mysql/resource-manager/readme.python.md [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+
+```yaml !$(track2)
+batch:
+  - tag: package-2020-01-01
+  - tag: package-2018-06-01
+  - tag: package-2017-12-01
+  - tag: package-2020-07-01-privatepreview
+  - tag: package-2020-01-01-privatepreview
+  - tag: package-2018-06-01-privatepreview
+  - tag: package-2017-12-01-preview
```
Comment (changlong-liu) [inline specification/mysql/resource-manager/readme.python.md:38]: Hi @mjain2 , want to make sure do we need so many tags?
I think there should be two: one for old server and one for flexibleServer.

File: specification/postgresql/resource-manager/readme.python.md [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+
+```yaml !$(track2)
+batch:
+  - tag: package-2020-01-01
+  - tag: package-2018-06-01
+  - tag: package-2017-12-01
+  - tag: package-2020-02-14-privatepreview
+  - tag: package-2020-01-01-privatepreview
+  - tag: package-2018-06-01-privatepreview
+  - tag: package-2017-12-01-preview
```
Comment (changlong-liu) [inline specification/postgresql/resource-manager/readme.python.md:38]: the same here, and those for track2

**Review by changlong-liu — CHANGES_REQUESTED** [review]
The tags change also include breaking change. Let's discuss the tags before merge

File: specification/postgresql/resource-manager/readme.python.md (hunk 2 above)
Comment (mjain2) [reply specification/postgresql/resource-manager/readme.python.md]: > The tags change also include breaking change. Let's discuss the tags before merge

Thanks Changlong.  Have fixed this with latest iteration.

**Review by changlong-liu — APPROVED** [review]

File: specification/mysql/resource-manager/readme.go.md [hunk 3]
```diff
@@ -5,25 +5,37 @@ These settings apply only when `--go` is specified on the command line.
...
 
+### Tag: package-2020-07-01-privatepreview and go
+
+These settings apply only when `--tag=package-2020-07-01-privatepreview --go` is specified on the command line.
+Please also specify `--go-sdk-folder=<path to the root directory of your azure-sdk-for-go clone>`.
+
+``` yaml $(tag) == 'package-2020-07-01-privatepreview' && $(go)
+rpname: mysql
+namespace: mysqlflexibleservers
+output-folder: $(go-sdk-folder)/services/preview/$(rpname)/mgmt/2020-07-01-preview/$(namespace)
```
Comment (ArcturusZhang) [inline specification/mysql/resource-manager/readme.go.md:29]: I am not quite sure if we could add this new variable here... could we just hard code this rpname in? like this:
```suggestion
namespace: mysql
output-folder: $(go-sdk-folder)/services/preview/mysql/mgmt/2020-07-01-preview/mysqlflexibleservers
```

**Review by ArcturusZhang — CHANGES_REQUESTED** [review]

**Review by ArcturusZhang — APPROVED** [review]
Please ignore the previous review, actually it works

**Pull request merged**